api/cluster.go

```diff
@@ -42,7 +42,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 	"go.uber.org/cadence/client"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I found three different naming of  k8s.io/apimachinery/pkg/apis/meta/v1, it would be fine to use one common name insted of metav1, v1meta and meta_v1